### DISCUSSION TITLE
Borg Backup Failure

### DISCUSSION THREAD

**Discussion by Aceriz** [nextcloud/all-in-one#3274] [Questions] [answered]
Hello All I am hoping that I Might be able to get some help, 

I had to have my UNRAID server offline for the last few weeks while I waited for and rebuild my array (which was successful) 

I was going through and updating all of the dockers including my NextcloudAIO. 

I wanted to run the BorgBackup before running the Update. But I am getting an error. 

```
Database export failed the last time. Most likely was the export time not high enough.
Cannot create a backup now.
Please report this to https://github.com/nextcloud/all-in-one/issues. Thanks!

```

I am able to run the backup integrity check and get a full pass as outlined below.  

Checking the backup integrity...
Starting repository check
finished segment check at segment 292
Starting repository index check
Index object count match.
Finished full repository check, no problems found.
Starting archive consistency check...
Starting cryptographic data integrity verification...
Finished cryptographic data integrity verification, verified 128794 chunks with 0 integrity errors.
Analyzing archive 20220830_052046-nextcloud-aio (1/6)
Analyzing archive 20220927_050039-nextcloud-aio (2/6)
Analyzing archive 20221015_052058-nextcloud-aio (3/6)
Analyzing archive 20221213_061732-nextcloud-aio (4/6)
Analyzing archive 20230118_070800-nextcloud-aio (5/6)
Analyzing archive 20230428_044934-nextcloud-aio (6/6)
Archive consistency check complete, no problems found.
Check finished successfully on 28.08.2023 - 22:23:07 (00 hours 46 minutes 24 seconds)

I am hoping that someone might be able to help me in figuring out the next steps in problem solving by when I attempt to run the BorgBackup I get the above error. 

Thanks,

**Comment by szaimen** [comment, marked as answer]
Hi, 
reason is that the database export failed the last time.

Most likely was the database container not correctly shut down via the AIO interface.
        
You might want to try the database export again manually by running the three commands:
```
sudo docker start nextcloud-aio-database
sleep 10
sudo docker stop nextcloud-aio-database -t 1800
```
        
Afterwards try to create a backup again and it should hopefully work.

If it should still fail, feel free to report this to https://github.com/nextcloud/all-in-one/issues and post the database container logs and the borgbackup container logs into the thread. Thanks!
